### DISCUSSION TITLE
[Tip] How to use PowerPoint slide layouts in Markdown

### DISCUSSION THREAD

**Discussion by noomorph** [jgm/pandoc#10190] [Show and tell]
Maybe it was just a stroke of bad luck, but... anyway, I earned this knowledge the hard way—I had to delve into pull requests, find out how you self-test with `*.native` files as input, convert them to Markdown, process them with an AI assistant to a "kitchen sink" format, and do some manual reviewing. Maybe this will save time for someone.

# Kitchen Sink

Here's the Markdown to PPTX conversion result _(disclaimer: opened and rendered via LibreOffice):_

![Preview markdown-to-pptx result, 15 slides](https://github.com/user-attachments/assets/a3340850-46af-4959-96bf-cb292d6d9f28)

## Sample code

```markdown
---
title: PowerPoint Kitchen Sink Presentation
author: Claude Assistant
date: September 17, 2024
---

# Introduction

## Welcome Slide

Welcome to our PowerPoint Kitchen Sink presentation!

::: notes
Remember to greet the audience warmly.
:::

---

# Section 1: Basic Layouts

## Title and Content

- This is a basic slide with bullet points
- It uses the "Title and Content" layout
- Perfect for simple content presentation

## Two Column Layout

::::: columns
::: column
Left column content:
- Point 1
- Point 2
:::
::: column
Right column content:
- Point A
- Point B
:::
:::::

## Comparison Layout

::::: columns
::: column
Text followed by an image:

![Moon](lalune.jpg)
:::
::: column
- This triggers the "Comparison" layout
- Useful for side-by-side comparisons
:::
:::::

## Content with Caption

Here's some explanatory text about the image below.

![The Moon](lalune.jpg "fig:")

# Section 2: Advanced Features

## Code Block

Here's a Python code block:

```python
def greet(name):
    return f"Hello, {name}!"

print(greet("World"))
``` 

## Table Example

| Column 1 | Column 2 | Column 3 |
|----------|----------|----------|
| Row 1    | Data     | More     |
| Row 2    | Info     | Stuff    |

## Incremental List

::: incremental
- This point appears first
- Then this one
- And finally this one
:::

## {background-image="lalune.jpg"}

::: notes
This is a slide with a background image and speaker notes only.
The "Blank" layout will be used.
:::

# Conclusion

## Thank You

Thank you for viewing this kitchen sink presentation!

::: notes
Remember to thank the audience and invite questions.
:::
```

While on stage with a microphone, I would like to express my gratitude to the Pandoc team and all the project contributors. 💌 

P. S. If this idea makes sense to you, maybe you could include some Markdown examples in your docs. E.g., this help article is helpful, but it lacks specific details: <https://pandoc.org/chunkedhtml-demo/10.1-structuring-the-slide-show.html>

**Comment by Saltsmart** [comment]
Thanks!

How could you render the Python code?

I've tried: `pandoc lalune.md -o lalune.pptx`, and it's:
<img width="927" height="542" alt="image" src="https://github.com/user-attachments/assets/f5a414a2-932e-47b7-8db8-b686d325bcd9" />

**Comment by DOZ-Michael** [comment]
Where I am struggling, what if don't have only those standard layouts, but you need to create your own layouts.
